CONTRIBUTING.md

```diff
@@ -17,6 +17,14 @@ The point of lspconfig is to provide the minimal configuration necessary for a s
 
 ## Adding a server to lspconfig
 
+When adding a new server configuration, please ensure that it meets one of the following criteria:
+
+- GitHub Stars: The server repository should have at least 100 stars.
```

Comment:
```suggestion
- GitHub Stars: The server repository should have at least 100 stars, or some other evidence (such as vscode marketplace downloads) that the LSP is reasonably popular.
    - This ensures that the configs in nvim-lspconfig are not quasi-marketing or spam.
```

Reply:
looks like there is duplicate the point 2 have mentioned the vscode marketplace